### DISCUSSION TITLE
明天黑神话悟空正式发售

### DISCUSSION THREAD

**Discussion by Pheathon** [share121/inter-knot#3078] [其他]
所以本人明天会暂停接取委托，顺便还要去直播游戏过程。
![下载](https://github.com/user-attachments/assets/bc36e33a-1a02-4ea1-88a2-74a37369b3dc)

**Comment by fairy-fire** [comment]
情感失败，明天必须给我接委托。<img src="https://bbs-static.miyoushe.com/static/2024/07/31/213614cd6275ec54d80fe9f84a0edf62_6334766189957723007.png" alt="白露 灭火" width="75" >不过话又说回来，要不把黑神话给我玩玩儿<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by Pheathon** [reply]
如果你愿意给我正常价格十倍的丁尼，那么我就会考虑你的要求了（笑）

**Comment by yzysrewrwqr** [comment]
风里雨里，空洞等你<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Reply by Pheathon** [reply]
千家玩家，绳匠回家
